pnpm-lock.yaml

```diff
@@ -284,10 +284,10 @@ importers:
     dependencies:
```

Comment:
`.lock` file blew up... I would still rather have the types of `@walletconnect/ethereum-provider`, than smaller lock file.